trino/base/trino-coordinator-dc.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apps/v1
```

Comment:
Please audit this file for config options that are already the defaults - we shouldn't define them here